packages/@pollyjs/core/tests/unit/test-helpers/mocha-test.js

```diff
@@ -4,7 +4,7 @@ class Sandbox {
   constructor(context) {
     this.beforeEachCalled = new Set();
     this.afterEachCalled = new Set();
-    this.context = context || { currentTest: {} };
+    this.context = context || { currentTest: { title: 'mockname' } };
```

Comment:
The test on line 28 (not visible) was the one that was now failing.  This was because the async was putting the [recording name assertion](https://github.com/Netflix/pollyjs/blob/master/packages/@pollyjs/core/src/utils/validators.js#L4) on the next tick.  I just needed to update the context object to give it a mock test name.

The other changes are me doing small clean ups on the test file for consistency.